src/main/app/windowManager.js

```diff
@@ -197,10 +220,73 @@ class WindowManager extends EventEmitter {
     return result
   }
 
+  /**
+   * Find the best window to open the files in.
+   *
+   * @param {string[]} fileList File full paths.
+   * @returns {{windowId: string, fileList: string[]}[]} An array of files mapped to a window id or null to open in a new window.
+   */
+  findBestWindowToOpenIn (fileList) {
+    if (!fileList || !Array.isArray(fileList) || !fileList.length) return []
+    const { windows } = this
+    const lastActiveEditorId = this.getActiveEditorId() // editor id or null
+
+    if (this.windowCount <= 1) {
+      return [ { windowId: lastActiveEditorId, fileList } ]
+    }
+
+    // Array of scores, same order like fileList.
+    let best = null
+    for (const window of windows.values()) {
+      if (window.type === WindowType.EDITOR) {
+        const score = window.getCandidatesScore(fileList)
```

Comment:
Can you rename `score` to other name, so that we can know it's an array, not a single score. Because he started to mislead me, I thought he was a single score.